tests/Doctrine/Tests/DBAL/Connections/MasterSlaveConnectionTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Doctrine\Tests\DBAL\Connections;
+
+use Doctrine\DBAL\Connections\MasterSlaveConnection;
+use Doctrine\DBAL\Driver;
+use Doctrine\Tests\DbalTestCase;
+
+class MasterSlaveConnectionTest extends DbalTestCase
+{
+    /**
+     * \Doctrine\DBAL\Connection::getParams has been marked @internal as of 2.11.0
+     * This test will be removed in 3.0
```

Comment:
this comment can be removed. MasterSlaveConnection itself has been deprecated and will be removed in 3.0 entirely anyway.

Reply:
removed